libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -24,10 +24,18 @@
       <input name="in1" type="vector2" nodename="N_mult_float"/>
       <input name="in2" type="vector2" interfacename="uvoffset"/>
     </subtract>
+    <divide name="N_divtilesize_float" type="vector2">
```

Comment:
Mostly changes to implementation additional scales for real-world units + some minor mods for separate and combine implementations / usage.